js/test/root/teststrings.js

```diff
@@ -4339,5 +4339,56 @@ module.exports.teststrings = {
 
         test.equal(str.formatChoice(4.1e2), "Default items");
         test.done();
+    },
+    testStringFormatChoiceGender: function(test) {
+        test.expect(2);
+        
+        var str = new IString("feminine#She is my friend|masculine#He is my friend|Default#They are my friends");
```

Comment:
Default -> "other" or just blank
"feminine#She is my friend|masculine#He is my friend|#They are my friends"